typescript/vropkg/src/parse/util.ts

```diff
@@ -31,7 +31,12 @@ export const xmlToTag = (tags): string[] => tags.children.filter(e => e.attr &&
 export const stringToCategory = (category): string[] => category.split(/(?<!\/)\./)
     .map(path => { return path.replace(/\//g, "") })
     .filter(path => path != "");
-
+export const validateWorkflowPath = (xmlDocument) => {
+	const invalidElement = xmlDocument.children.find((e) => e.attr.name == null || e.attr.name =='undefined');
+	if (invalidElement && invalidElement.firstChild!=null){
+		console.warn(`Unsupported characters detected in the workflow path. The value "${invalidElement.firstChild.val}" contains characters that are not allowed. The folders with special characters will be ignored.`);
```

Comment:
I think it would be better to throw an error in this case, as it would give the user a hint why the packaging / unwrapping did not succeed.

Reply:
Thanks for the review. Initially, I throw an error in this case. But later found that with "JS vRO" type projects with similar case the folder is excluded and pull operation getting success. So to keep the system behavior consistent, we thought to add warning message in both cases and make pull operation success, excluding the special character folder.
Yes, we can go with throwing error on both types as well. WDYT @Michaelpalacce   ?

Reply:
Further reflection, I've realized that my previous idea was misguided. Without throwing the error, the process of pulling and then pushing the project would miss the special character folder on the server. So the implementation added for throwing the error.